shesmu-server-ui/src/histogram.ts

```diff
@@ -25,7 +25,7 @@ export interface HistogramRow {
    */
   selectionDisplay(start: number, end: number): string;
   /**
-   * A callback inidcating that the user has selected this range of the histogram.
+   * A callback inidicating that the user has selected this range of the histogram.
```

Comment:
"indicating"

Reply:
Fixed in 48916bcb8b640282890da1e894e5635bad6c3531